### PR TITLE
Cart page

### PR DESCRIPTION
**Pull request by rahulyadav139**
I have created cart page for the e-com site. Kindly review it.

### CODE REVIEW

**Review by j24m — APPROVED** [review]

File: cart.html [hunk 1]
```diff
@@ -0,0 +1,256 @@
...
+              <button class="btn icon small">
+                <i class="fas fa-minus-circle"></i>
+              </button>
+              <div class="flex center">5</div>
+              <button class="btn icon small">
+                <i class="fas fa-plus-circle"></i>
+              </button>
+            </div>
+
+            <div class="cart-item__buttons">
```
Comment (tarunsankhla) [inline cart.html:219]: classname dash or snake follow one ? Iknow I have comment on other PR also same just make changes here also rest everything lookg good

**Review by tarunsankhla — APPROVED** [review]

File: product-detail.css [hunk 2]
```diff
@@ -0,0 +1,29 @@
+.card.product-detail {
+  display: grid;
+  grid-template-columns: 4fr 6fr;
+  grid-template-rows: 19rem;
+  box-shadow: none;
+}
+.product-detail .image {
+  /* padding: 0 5rem; */
```
Comment (aiman-ansari) [inline product-detail.css:8]: remove comments

File: product-detail.css [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+  object-fit: cover;
+  object-position: center;
+}
+.main {
+  width: 100%;
+  max-width: 1000px;
+
+  margin: 1rem auto;
+  padding: 1rem;
+  /* border: 1px solid red; */
```
Comment (aiman-ansari) [inline product-detail.css:25]: remove spacing as well as comment also

File: wishlist.css [hunk 4]
```diff
@@ -0,0 +1,22 @@
+.main {
+  width: 100%;
+  max-width: 1000px;
```
Comment (aiman-ansari) [inline wishlist.css:3]: convert it into rem

File: cart.html [hunk 5]
```diff
@@ -0,0 +1,256 @@
...
+      <div class="brand-logo">
+        <i class="fas fa-angle-left"></i><i class="fas fa-globe"></i
+        ><i class="fas fa-angle-right"></i>
+      </div>
+
+      <div class="input-field-icon">
+        <label
+          ><span class="icon small"><i class="bi bi-search"></i></span
+          ><input placeholder="Search" type="text"
+        /></label>
```
Comment (aiman-ansari) [inline cart.html:27]: your code indentation is not proper here maybe because of your code editor

**Review by aiman-ansari — APPROVED** [review]
- Code is very clean 
- Variables naming are meaningful.
- Your UI looks so good.

File: cart.html (hunk 5 above)
Comment (rahulyadav139) [reply cart.html:27]: I am unable to remove this issue. I tried but let me change my code formatter.

File: wishlist.css (hunk 4 above)
Comment (rahulyadav139) [reply wishlist.css:3]: I forgot this. I'll change this.

**Pull request merged**